devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -233,13 +248,60 @@ private getSupportedButtonValues() {
 private getModelNumberOfButtons() {[
 		"3450-L" : 4,
 		"3450-L2" : 4,
-		"SceneSwitch-EM-3.0" : 4
+		"SceneSwitch-EM-3.0" : 4, 
+		"ADUROLIGHT_CSC" : 4,
+		"Adurolight_NCC" : 4
 ]}
 
 private getModelBindings(model) {
 	def bindings = []
 	for(def endpoint : 1..modelNumberOfButtons[model]) {
 		bindings += zigbee.addBinding(zigbee.ONOFF_CLUSTER, ["destEndpoint" : endpoint])
 	}
+	if (isAduroSmartRemote()) {
+		bindings += zigbee.addBinding(zigbee.LEVEL_CONTROL_CLUSTER, ["destEndpoint" : 2]) + 
+			zigbee.addBinding(zigbee.LEVEL_CONTROL_CLUSTER, ["destEndpoint" : 3])
+	}
 	bindings
 }
+
+private Map parseAduroSmartButtonMessage(Map descMap){
+	def buttonState = "pushed"
+	def buttonNumber = 0
+	if (descMap.clusterInt == zigbee.ONOFF_CLUSTER) {
+		if (descMap.command == "01") {
+		    buttonNumber = 1
+		} else if (descMap.command == "00") {
+		    buttonNumber = 4
+		} else {
+		    return [:]
+		}
+	} else if (descMap.clusterInt == zigbee.LEVEL_CONTROL_CLUSTER) {
+		if (descMap.command == "02") {
+		    def data = descMap.data
+		    def d0 = data[0]
+		    if (d0 == "00") {
+			buttonNumber = 2
+		    }else if (d0 == "01") {
+			buttonNumber = 3
+		    }
+		}
+	} else if (descMap.clusterInt == ADUROSMART_SPECIFIC_CLUSTER) {
+		def list2 = descMap.data
+		buttonNumber = (list2[1] as int) + 1
+	}
+	if (buttonNumber != 0) {
+		def childevent = createEvent(name: "button", value: "pushed", data: [buttonNumber: 1], isStateChange: true)
+		sendEventToChild(buttonNumber, childevent)
+		def descriptionText = "$device.displayName button $buttonNumber was $buttonState"
+		return createEvent(name: "button", value: buttonState, data: [buttonNumber: buttonNumber], descriptionText: descriptionText, isStateChange: true)
+        } else {
```

Comment:
nit: spacing

Reply:
Thank you review.
In these few lines of code, I didn't find any problems with word spacing. Maybe I didn't see it.